### ISSUE TITLE
batman does not respect PAGER or MANPAGER with `--paging=never` in bat's config

### ISSUE THREAD

**Issue by benjamb** [eth-p/bat-extras#128]
I would have expected `batman` to still make use of `$PAGER` even if `--paging=never` was set in `~/.config/bat/config`.

This may be an unreasonable expectation, but I wanted to open this up for discussion.

**Comment by eth-p** [comment]
Good catch. I'd like some more opinions on this before making a change to the default behavior, though. The current implementation is more consistent with `bat`, but that comes at the cost of being less consistent with `man`.

In the meantime, you can actually set `--paging=always` as part of the `batman --export-env` command, and that will pass it along when `bat` is called.
